lightningd/pay.c

```diff
@@ -42,6 +42,34 @@ struct sendpay_command {
 	struct command *cmd;
 };
 
+static bool string_to_payment_status(const char *status_str, enum wallet_payment_status *status)
+{
+	if (streq(status_str, "complete")) {
+		*status = PAYMENT_COMPLETE;
+		return true;
+	} else if (streq(status_str, "pending")) {
+		*status = PAYMENT_PENDING;
+		return true;
+	} else if (streq(status_str, "failed")) {
+		*status = PAYMENT_FAILED;
+		return true;
+	}
+	return false;
+}
+
+static const char *payment_status_to_string(const enum wallet_payment_status status)
+{
+	switch (status) {
+	case PAYMENT_COMPLETE:
+		return "complete";
+	case PAYMENT_FAILED:
+		return "failed";
+	default:
```

Comment:
it's best practice not to use `default` for a switch statement. instead, name all of the cases.